src/hw_bridge.cpp

```diff
@@ -3,43 +3,3 @@
 #else
 #include "hw_bridge.h"
 #endif
-
-std::string HWBRIDGE::ARM::ACTUATOR::stringifyActuatorID(HWBRIDGE::ARM::ACTUATOR::ActuatorId actuator) {
-  switch (actuator) {
-    case (HWBRIDGE::ARM::ACTUATOR::TURNTABLE):
-      return "TURNTABLE";
-    case (HWBRIDGE::ARM::ACTUATOR::SHOULDER):
-      return "SHOULDER";
-    case (HWBRIDGE::ARM::ACTUATOR::ELBOW):
-      return "ELBOW";
-    case (HWBRIDGE::ARM::ACTUATOR::WRISTLEFT):
-      return "WRISTLEFT";
-    case (HWBRIDGE::ARM::ACTUATOR::WRISTRIGHT):
-      return "WRISTRIGHT";
-    case (HWBRIDGE::ARM::ACTUATOR::CLAW):
-      return "CLAW";
-    default:
-      return "ERROR";
-  }
-}
-
-std::string HWBRIDGE::ARM::PID::stringifyParam(HWBRIDGE::ARM::PID::Parameter param) {
-  switch (param) {
-    case (HWBRIDGE::ARM::PID::P):
-      return "P";
-    case (HWBRIDGE::ARM::PID::I):
```

Comment:
i created these functions way back when i was working on the pid api and merged it in before the pid api stuff was ever merged in